ma/ma.cc

```diff
@@ -115,6 +120,11 @@ void adaptVerbose(Input* in, bool verbose)
   apf::printStats(m);
 }
 
+void adaptVerbose(const Input* in, bool verbose)
+{
+  adaptVerbose(makeAdvanced(in), verbose);
+}
+
```

Comment:
make `Adapt` take `const Input*` and this function becomes unnecessary.

Reply:
@jacobmerson mesh adapt changes the input internally. So I will need both versions of these functions.